### ISSUE TITLE
patch_id=52 starts at the wrong date from queries

### ISSUE THREAD

**Issue by hainej96** [odota/web#3089]
**Current Behavior**
I am using patch_id=52 (7.33) in my queries to return win rates from different patches, I have notice that it has not included the first few games i played for the first week of the patch and has instead put them into patch_id=51. 

**Expected behavior/code**
Should return the correct schema for the patch using patch_id=?. Maybe the dates might be off? 

**Environment**
- Browser Version: [Latest Chrome ]
- OS: [Windows 10]

**Possible Solution**
Not sure if its because of the integer string in the schema for "patch_id" displays incorrectly at the start of every new patch could instead maybe use the date that the new patch was launched? i.e >20/04/2023 17:00 PST = 52??

**Comment by howardchung** [comment]
Yeah, this is due to a delay in updating the backend--it's not super easy to back-correct the data so I'm hoping it'll just be less noticeable as the patch goes on
